app/views/shared/_review-crr.html.erb

```diff
@@ -0,0 +1,45 @@
+<div id="reviews" class="carousel slide" data-ride="carousel">
+
+    <div class="carousel-inner">
+        <div class="item active">
+
+            <div class="col-md-10 col-md-offset-1">
+
+                <h4><i class="fa fa-quote-left"></i>Lorem ipsum dolor sit amet, consectetur adipiscing Lorem ipsum dolor sit amet, consectetur adipiscing elit onec molestie non sem vel condimentum. <i class="fa fa-quote-right"></i></h4>
+                <div class="user-img pull-right">
+                    <%# <img src="assets/img/user.gif" alt="" class="img-u image-responsive" /> %>
+                </div>
+                <h5 class="pull-right"><strong class="c-black">Lorem Dolor</strong></h5>
+            </div>
+        </div>
+        <div class="item">
+            <div class="col-md-10 col-md-offset-1">
+
+                <h4><i class="fa fa-quote-left"></i>Lorem ipsum dolor sit amet, consectetur adipiscing Lorem ipsum dolor sit amet, consectetur adipiscing elit onec molestie non sem vel condimentum. <i class="fa fa-quote-right"></i></h4>
```

Comment:
what is this.

Reply:
Sample carousel that will be populated with some comments by customer